zdm/survey.py

```diff
@@ -32,7 +32,6 @@ class Survey:
 
```

Comment:
Just realized here that this branch has not pulled in `main` yet,
so I am going to stop reviewing until we do.